changelog.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## Version 3.1.2 [03-April-2018]
+
+### Added
+- Made trigger of auto renew URL job configurable. Developer can disable the execution of the job from the installing app.
```

Comment:
Functionality to disable URL renew job for installing application.

Reply:
Changed